src/steps/findings.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  createDirectRelationship,
+  createIntegrationEntity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  RelationshipClass,
+  IntegrationMissingKeyError,
+} from '@jupiterone/integration-sdk-core';
+
+import { createAPIClient } from '../client';
+import { IntegrationConfig } from '../types';
+import { getVulnerabilityLink, getVulnerabilityNumber } from '../util';
+
+export async function fetchFindings({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+
+  await apiClient.iterateFindings(async (finding) => {
+    const findingProps = finding.resource;
+    const findingEntity = await jobState.addEntity(
+      createIntegrationEntity({
+        entityData: {
+          source: finding,
+          assign: {
+            _type: 'cobalt_finding',
+            _class: 'Finding',
+            _key: findingProps.id,
+            tag: findingProps.tag,
+            name: findingProps.title,
+            displayName: findingProps.title,
+            description: findingProps.description,
+            category: 'Penetration Testing',
+            typeCategory: findingProps.type_category,
+            labels: JSON.stringify(findingProps.labels, null, 2),
```

Comment:
Take a look at `assignTags` in the SDK. There was a very recent PR in the graph-google-cloud project that converts `labels` there to `tag.LABEL: VALUE` properties. These are rendered with a tag icon in the J1 UI.

Reply:
Cool feature. Currently, our test data does not have any `labels` in Findings, and I can't add more myself, but I've implemented the feature, and once we see it on the live data of JupiterOne account, we might tweak how it works.